pkg/controller/association/controller/beat_kibana.go

```diff
@@ -68,12 +72,40 @@ func getBeatKibanaRoles(associated commonv1.Associated) (string, error) {
 		)
 	}
 
+	if strings.Contains(beat.Spec.Type, ",") {
+		return "", fmt.Errorf("beat type %s should not contain a comma", beat.Spec.Type)
+	}
+
 	v, err := version.Parse(beat.Spec.Version)
 	if err != nil {
 		return "", err
 	}
-	if v.IsSameOrAfter(version.From(7, 5, 0)) {
-		return "kibana_admin", nil
+
+	// Roles for supported Beats are based on:
+	// https://www.elastic.co/guide/en/beats/filebeat/7.6/feature-roles.html#privileges-to-setup-beats
+	// Docs are the same for all Beats. For a specific version docs change "current" to major.minor, eg:
```

Comment:
The example above does not contain current, so maybe just change this text to "change the major.minor version in the URL"?

Reply:
Replaced with current as this is what we have in `beat_es.go`.